src/main/java/model/Car.java

```diff
@@ -0,0 +1,73 @@
+package model;
+
+public class Car {
+    private static final int DRIVE_POSITION = 1;
+    private static final int CONDITION_MIN_VALUE = 0;
+    private static final int CONDITION_MAX_VALUE = 10;
+    private static final int CONDITION_GOOD_MIN_VALUE = 4;
+    private static final boolean GOOD = true;
+    private static final boolean BAD = false;
+    private static final int CORRECT_NAME_LENGTH_LIMIT = 5;
+    private static final String NOT_SUITABLE_INPUT = "부적합한 입력입니다.";
+    private static final String CAR_POSITION_VIEW = "-";
+
+    private final String name;
+    private int position = 0;
+
+    public Car(String name) {
+        validateName(isCorrectName(name));
+        this.name = name;
+    }
+
+    private void validateName(boolean isCorrectName) {
+        if (!isCorrectName) {
+            throw new IllegalArgumentException(NOT_SUITABLE_INPUT);
+        }
+    }
+
+    private boolean isCorrectName(String name) {
+        return isNotBlank(name) && isLengthIsBelowCorrectNameLengthLimit(name);
+    }
+
+    private boolean isNotBlank(String name) {
+        return !name.contains(" ");
```

Comment:
```suggestion
        return !name.isEmpty();
```
이렇게 수정하는게 더 좋지 않을까요?

Reply:
제시해주신 피드백은 감사드립니다. 다만, `isEmpty()`와 `contains(" ")`는 다른 의미를 갖고 있다고 생각됩니다.
`isEmpty()`의 경우에는 `""`과 같이 아무 값이 없을때를 뜻하고 `contains(" ")`는 문자열 사이에 공백의 포함유무를 체크하는 것이라 서로 다르다고 생각합니다. 예를 들어보면 `"   hi"`와 `"    "`는 다른 것처럼 말입니다. 이 경우에는 `isEmpty()`를 사용하려면 `name.trim().isEmpty()`를 사용하는 것이 적절하지 않은가 라고 생각되는데 의견이 궁금합니다.

Reply:
이부분은 제가 이전 코드에서 입력값을 trim() 함수로 입력을 받는다고 착각해서 잘못 전달을 드린 것 같습니다. 죄송합니다 🙇

Reply:
꼼꼼하게 봐주셔서 감사합니다.